pydarn/plotting/power.py

```diff
@@ -0,0 +1,146 @@
+#  Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+#  Author: Cooper Ross Robertson, Summer Student 2020
+import matplotlib.pyplot as plt
+import numpy as np
+
+from typing import List
+
+from pydarn import SuperDARNRadars, exceptions, RTP
+
+
+class Power():
+    """
+    Power class to plot SuperDARN RAWACF data power related.
+
+
+    Methods
+    -------
+    plot_lag0
+    """
+    def _str_(self):
+
+        return "This class provides the following methods: \n"\
+                " - plot_interference()"
```

Comment:
Agree that a rename is a good idea; just a leftover reference to the original name here. I think we may need to differentiate this from a range-time plot, not sure if it's necessary in the name or just in the description though.
```suggestion
                " - plot_lag0()"
```

Reply:
is `plot_lag0` proper or `plot_politscan_noise`? Kind of ignorant of this side of things.

Reply:
I think that although it was helpful for politescan data, technically there is nothing stopping someone from using other data, so I would say to avoid using politescan in the name. I think that mentioning the politescan use case in the notes is good enough. 

To be more indicative of what's happening inside and since it is specifically plotting a 'pwr0' vector statistic ('pwr0' parameter hardcoded in the function), my preference would be to choose something closer to that in the name. I think plot_lag0 is alright, or potentially plot_pwr0_statistic ?

Reply:
changed the name